java/src/test/android/app/src/main/AndroidManifestQnn.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <application
+        android:allowBackup="true"
+        android:icon="@mipmap/ic_launcher"
+        android:label="@string/app_name"
+        android:roundIcon="@mipmap/ic_launcher_round"
+        android:supportsRtl="true"
+        android:theme="@style/Theme.JavaValidator">
+        <activity android:name=".MainActivity" android:exported="true">
+            <intent-filter>
+                <action android:name="android.intent.action.MAIN" />
+
+                <category android:name="android.intent.category.LAUNCHER" />
+            </intent-filter>
+        </activity>
+        <uses-native-library
```

Comment:
Any suggestions on handling  uses-native-library block without adding a new file? I tried Manifest Placeholders but it does not seem to work with complete element. I also tried sending the nativeLibrary value with manifest placeholder but an empty value for the non QNN case does not work

Reply:
I don't know of a way, but it's as test app so a duplicate manifest doesn't seem like a huge issue. A small ReadMe.md in the directory explaining why they exist would be useful though.